envoy_pkg/.bazelrc

```diff
@@ -90,3 +90,5 @@ build:linux-glibc-docker --experimental_enable_docker_sandbox
 
 # This is no-op but if this doesn't exist bazel will error with non-exist config
 build:darwin --announce_rc
+build:darwin --config=libc++
```

Comment:
those are no-op or breaking stuff, macOS only have libc++ and they are default

Reply:
ah, that's right...